# Define the goal
float32 tool_error_threshold
---
# Define the result
uint32 result
---
# Define a feedback message
float32 servo_error